test/models/delivery_detail.py

```diff
@@ -1,23 +1,212 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
+# Part of Odoo and Trescloud. See LICENSE file for full copyright and licensing details.
 
-from odoo import models, fields, api
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
 
 
 class DeliveryDetail(models.Model):
     _name = 'delivery.detail'
+    _inherit = ['mail.thread', 'mail.activity.mixin']
```

Comment:
```suggestion
    _inherit = ['mail.thread']
```
Solo como nota
No es necesario para el tracking.